pkg/models/repository_configuration.go

```diff
@@ -44,5 +48,16 @@ func (rc *RepositoryConfiguration) BeforeCreate(tx *gorm.DB) (err error) {
 	if rc.OrgID == "" {
 		err = Error{Message: "Org ID cannot be blank.", Validation: true}
 	}
+
+	if rc.Arch != "" && !config.ValidArchLabel(rc.Arch) {
+		err = Error{Message: fmt.Sprintf("Specified Distribution Architecture %s is invalid.", rc.Arch),
```

Comment:
Small thing, but I think there should be quotes around %s, e.g. "Specified Distribution Architecture "bad arch" is invalid."

Reply:
yes, that is not a title, just two words in a sentence, so lower case for Distribution Architecture". This also need another rebase.